upgrade/scripts/upgrade/csm-upgrade.sh

```diff
@@ -156,6 +156,9 @@ fi
 # Restart CFS deployments to avoid CASMINST-6852
 "${basedir}/../common/restart-cfs.sh"
 
+# Update test/CLI RPMs on NCNs
```

Comment:
Should a state be tracked for this? It won't hurt to do it multiple times but I imagine it would be best to only do it once. Specifically I mean doing the following

``bash
state_name="UPDATE_TEST_CLI_RPMS"
#shellcheck disable=SC2046
state_recorded=$(is_state_recorded "${state_name}" $(hostname))
if [[ $state_recorded == "0" ]]; then
  echo "====> ${state_name} ..."
  ...
```

Reply:
Sure, I'm fine adding that. I forget why I chose not to do it in the first place.